src/plugins/sorting-task/styles.css

```
@@ -0,0 +1,12 @@
+.question-card{
+  padding: 32px;
+  border: 1px solid lightgray;
+  position: relative;
+}
+.question-card:after{
```

```suggestion
.question-card::after {
```

`:after` (with a single colon) has been used in the past. However, in order to distinguish between pseudo-selectors and actual pseudo-elements, a double colon should be used for the latter.